CHANGELOG.md

```diff
@@ -7,4 +7,23 @@ First public release, converts VCF variants into a [FHIR Genomics Diagnostic Rep
 * Supports VCF file (text-based or bgzipped) and optionally tabix files for query.
 * Supports genome build ('GRCh37' or 'GRCh38');
 * Optionally supports a query region in the form of .bed or dictionary that indicates the region(s) to convert.
-* Optionally supports inclusion of  'region-studied' observations that detail which portions of the conversion region were studied, and of those studied regions, which portions were deemed uncallable.
\ No newline at end of file
+* Optionally supports inclusion of  'region-studied' observations that detail which portions of the conversion region were studied, and of those studied regions, which portions were deemed uncallable.
+
+# v0.0.17
+
+## Added
+* Exclusion criteria, for cases where VCF CHROM is not recognized.
+* A configurable parameter for mitochondrial DNA homoplasmic vs heteroplasmic assignment.
+* Data Type Validation for 'has_tabix' and 'ratio_ad_dp'.
+* Github Actions to run unit tests and PEP8 validation tests against any new PR that is raised.
+* Pull Request template, Issue template, and Code of Conduct.
+* Minimum support version check for pyranges.
+* A utility function to create Ordereddict.
+
+## Changed
+* Refactored python code so that it conforms to PEP8.
+* Refactored tests to separate unit tests and integration tests.
+* Refactored 'Common.py' to remove utility class and create utility functions.

```

Comment:
There is no need to include refactoring details in Changelog as long as it does not change any functionality. Changelog file are mostly for the user and they don't care what happens to code, there care about features added, deleted, fixed, etc.
```suggestion

```